dnd5esheets/front/package.json

```diff
@@ -5,12 +5,13 @@
     "build": "vite build",
     "start": "vite",
     "serve": "vite preview",
-    "generate-client": "openapi --input ./openapi.json --output ./src/5esheets-client",
+    "generate-client": "openapi --useUnionTypes --input ./openapi.json --output ./src/5esheets-client",
     "generate-effect-parser": "peggy --plugin ./node_modules/ts-pegjs/dist/tspegjs -o src/effects/parser/index.ts src/effects/parser/grammar.pegjs",
     "test": "vitest --run",
     "lint": "eslint src",
     "prettier": "prettier -w .",
-    "prettier-check": "prettier -c ."
+    "prettier-check": "prettier -c .",
+    "typecheck": "tsc --noEmit"
```

Comment:
Should we add this to `make front-check` ?

Reply:
```
Found 23 errors in 3 files.

Errors  Files
    17  src/effects/index.ts:29
     5  src/effects/parser/index.test.ts:3
     1  src/effects/parser/index.ts:119
```

Maybe in another PR

Reply:
Precisely 😬 
I spent some time trying to type the effect parser, but couldn't get to a satisfying solution yet. I'll look at it again later.
In the man time, we could just ignore this file and add typecheck to `make front-check`.